src/Microsoft.ML.StandardTrainers/Microsoft.ML.StandardTrainers.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
```

Comment:
@ericstj, could you please take a look? I am not quite familiar with the build settings. Thank you!